### DISCUSSION TITLE
Google CLA check -- available outside of Google?

### DISCUSSION THREAD

**Discussion by Symbolics** [google/docsy#743] [Q&A] [answered]
Is integration with Google CLA check available outside of Google? The use case is public contributions to a Docsy repository.

I found [cla-assistant](https://github.com/cla-assistant/cla-assistant), which looks like it will do the job, but it would be easier to use the Google CLA checker (I think?) because that's what Docsy is using now. At least the hope is to reuse the integration scripts.

**Comment by emckean** [comment, marked as answer]
Hi! Only Google projects can use the Google CLA (which gives a license to the contributions to Google, which your project may not want :) )

If you're looking for a CLA tool for your project, you might want to look into the Developer Certificate of Origin (tool to enforce: https://probot.github.io/apps/dco/)  or maybe the Linux Foundation's EasyCLA (https://easycla.lfx.linuxfoundation.org/)
